tokio/src/sync/broadcast.rs

```diff
@@ -1521,7 +1521,7 @@ impl<T: Clone> Receiver<T> {
     /// context.
     ///
     /// # Examples
-    /// ```
+    /// ```ignore-wasm
```

Comment:
These kind of test can also be enabled partially, see https://github.com/tokio-rs/tokio/blob/5f3f5b0be4ec933aed2290ea9d929b5daadae321/tokio/src/net/udp.rs#L2137.